src/html_toast.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Blockly from 'blockly/core';
+
+/**
+ * Configuration options for toasts.
+ */
+interface HtmlToastOptions extends Blockly.ToastOptions {
+  element?: HTMLElement;
+}
+
+/**
+ * Custom toast implementation that supports HTML elements in toast messages.
```

Comment:
This might be worth updating to include an example of how to use `HtmlToast` since it seems that the options object needs to be very particularly set up in order for this to work correctly.

Reply:
Done.